src/ui/Radio/Radio.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import { Root, Item, Indicator } from "@radix-ui/react-radio-group";
+
+import { Typography } from "../Typography";
+
+import {
+  radioItemStyle,
+  radioIndicatorStyle,
+  itemLabelBox,
+  radioRootStyle,
+} from "./Radio.css";
+
+interface RadioProps {
+  defaultValue?: string;
+  disabled?: boolean;
+  onValueChange?: (e?: string) => void;
+  radioValue: string[];
+}
+
+const Radio = ({
+  radioValue,
+  disabled = false,
+  onValueChange,
+  defaultValue,
+}: RadioProps) => (
+  <Root
+    aria-label="View density"
+    className={radioRootStyle}
+    defaultValue={defaultValue}
+    onValueChange={onValueChange}
+  >
+    {radioValue.map((radioData, index) => (
+      <div className={itemLabelBox} key={radioData}>
+        <Item
+          className={radioItemStyle}
+          disabled={disabled}
+          id={String(index)}
+          value={radioData}
+        >
+          <Indicator className={radioIndicatorStyle} />
+        </Item>
+        <label htmlFor={String(index)}>
+          <Typography fontSize="body2-m">{radioData}</Typography>
+        </label>
```

Comment:
```suggestion
         <Typography as="label" fontSize="body2-m">{radioData}</Typography>
```
이렇게하면 불필요하게 태그를 두단계로 만들지 않아도 될 것 같아요~!